joint_state_controller:

    type: joint_state_controller/JointStateController
    publish_rate: 100

joint_position_pd_gravity_controller:

    type: franka_sim_controllers/JointPositionPDGravityController
    state_publish_rate: 100

    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
